### DISCUSSION TITLE
#32 togETHer meeting - Feb 7 2022

### DISCUSSION THREAD

**Discussion by bobjiang** [tog-ETH-er/togETHer#7] [General]
Agenda:

Community Updates
Fireside discussion

会议主题：togETHer
会议时间：2020/12/28 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

手机一键拨号入会
+8675536550000,,89150986775#(中国大陆)
+85230018898,,,2,89150986775#(中国香港)

根据您的位置拨号
+8675536550000(中国大陆)
+85230018898(中国香港)

**Comment by yanyanho** [comment]
### Dapp-Learning

#### 翻译小组
MIT公开课翻译第9/10集进入审稿阶段
#### 行研群
建设行研群，一起读messari，dephi研报；

**Comment by luckyyang** [comment]
## Rebase
- Web3 Daily [746-759 期](https://rebase.network/posts/category/web3daily) 
- Rostra [Sprint 4 开发](https://rostra.notion.site/Sprint-4-a3fe936e8dec4b77b10775342e6af879)

**Comment by Riely007** [comment]
### 以太坊翻译小组

- 官网翻译进行中

**Comment by winkybupt** [comment]
## 原语里弄
1. Lane’s Choice 日历已经发布  https://github.com/PrimitivesLane/Lanes-Choice-Calendar

**Comment by bobjiang** [comment]
- ENS ecosystem (integration)
- GitcoinDAO onboarding re-open
- Kernel Block 5

**Comment by Emilyxiannv** [comment]
### DAOSquare
邀请Plancker DAO 的Cliff 做了一期关于Gamefi的分享，链接：https://www.youtube.com/watch?v=WmXPuEJz-8M

**Comment by Jackieyewang** [comment]
筹备第一次crypto speak low，话题是2022年DEFI的发展趋势
